tools/devops/automation/templates/api-diff-stage.yml

```diff
@@ -25,7 +25,7 @@ parameters:
 
 - name: xcodeChannel
   type: string
-  default: Stable
+  default: Beta
```

Comment:
```suggestion
  default: Stable
```

Reply:
Fixed to not set a default value in the first place.